This is Simulation Version 4.20 and Hardware Version 4.20.

Program called with 2 arguments: z502.exe test1a
Calling with argument 'sample' executes the sample program.
Simulation is running as a UniProcessor
Add an 'M' to the command line to invoke multiprocessor operation.

PID: 1, process_context: 8759800
This is Release 4.20:  Test 1a
SVC handler: get_time 
Arg 0: Contents = (Decimal) 11665012,  (Hex)   B1FE74
Time of the day: 36
SVC handler: sleep    
Arg 0: Contents = (Decimal)     3000,  (Hex)      BB8
in start timer
Got expected result (DEVICE_IN_USE) for Status of Timer

The next output from the Interrupt Handler should report that 
   interrupt of device 4 has occurred with no error.
halt started
halt ended
Interrupt_handler: Found device ID 4 with status 0
8759800 is being popped out of the timer queue
process popped out of timer queue successfully
interrupt handler completed
SVC handler: get_time 
Arg 0: Contents = (Decimal) 11665000,  (Hex)   B1FE68
Time of the day: 3063
Sleep Time = 3000, elapsed time= 3027
SVC handler: term_proc
Arg 0: Contents = (Decimal)       -1,  (Hex) FFFFFFFF
Arg 1: Contents = (Decimal) 11665028,  (Hex)   B1FE84

Hardware Statistics during the Simulation
Context Switches =     1:  Mean Number Running Threads =   1.00   CALLS =    22
  The Z502 halts execution and Ends at Time 3074
Exiting the program
